Sources/Turf/Turf.swift

```diff
@@ -46,3 +46,21 @@ public func mid(_ coord1: CLLocationCoordinate2D, _ coord2: CLLocationCoordinate
     let heading = coord1.direction(to: coord2)
     return coord1.coordinate(at: dist / 2, facing: heading)
 }
+
+/**
+ Takes a coordinate and calculates the circular polygon given a radius in meters,
+ using a defined number of steps for precision.
+
+ - Parameter center: The center coordinate for the circle.
+ - Parameter radius: The radius of the circle, measured in meters.
+ - Parameter steps: The number of vertices the circle will have. Defaults to 64.
+ */
+public func circle(from center: CLLocationCoordinate2D, radius: CLLocationDistance, steps: Int = 64) -> Polygon {
+
+    let coordinates = (0...steps).map { ( step ) -> CLLocationCoordinate2D in
+        let bearing = CLLocationDirection(step * -360 / steps)
```

Comment:
Is there a particular reason for going counterclockwise? Technically, CLLocationDirection uses a negative value to indicate an invalid direction. It looks like we’re getting away with it here, but there’s no guarantee that a future iteration of `coordinate(at:facing:)` won’t require the direction to be positive.

Reply:
If you do end up following the Objective-C implementation in https://github.com/mapbox/turf-swift/pull/123#discussion_r532734132, it may be necessary to go counterclockwise to patch up the polygon in the right order. Either way, it would be safer to mod the angle to [0, 360) to avoid a negative angle.

Reply:
>Is there a particular reason for going counterclockwise?

I was closely following the [JS implementation](https://github.com/Turfjs/turf/blob/17002ccd57e04e84ddb38d7e3ac8ede35b019c58/packages/turf-circle/index.ts#L36) :) 

Would `fabs()` around the bearing be sufficient? That keeps everything in a positive value. I've included it in the latest change.

Reply:
Yes, that will effectively make it go clockwise.